index.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>HTML QUIZ</title>
+	<link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+<div class="card">
+	<article>
+		<header>
+			<h1 class="lineGray">Blueberry Pie</h1>
+		</header>
+			<img class="imgPick" src="http://3.bp.blogspot.com/-VGv-McF4Fgg/T8aW29BfmHI/AAAAAAAACzo/gRzyusPyYfE/s1600/IMG_2941.jpg" alt="Blue">
```

Comment:
The `<img>` tag is kind of a weirdo. It displays a combination of block and inline properties. To meet the quiz requirements it would be best to wrap the `<img>` tag inside a `<div>` tag.